internal/rbd/controllerserver.go

```diff
@@ -161,6 +161,18 @@ func buildCreateVolumeResponse(ctx context.Context, req *csi.CreateVolumeRequest
 	return &csi.CreateVolumeResponse{Volume: volume}, nil
 }
 
+func getGRPCError(err error) error {
```

Comment:
This function should probably not be used to convert any error to a GRPCError? Please clearly comment the intention so that when a new error gets added to the conversion, the comment gets updated too.